### DISCUSSION TITLE
butterfly Website

### DISCUSSION THREAD

**Discussion by Sarraj-Sahar** [LinwoodDev/Butterfly#359] [Q&A] [answered]

**Comment by CodeDoctorDE** [comment, marked as answer]
> Hi, I wanted to ask if this open-source project includes the code for the Butterfly website (https://docs.butterfly.linwood.dev/) or if it's just for the Butterfly app. If yes I have noticed some issues on the website that I would like to point out (create issues for them).

It's already opensource: https://github.com/LinwoodCloud/Butterfly/tree/develop/docs
